thesis-gui/src/types/my-quiz.ts

```diff
@@ -24,3 +24,14 @@ export type QuestionsSetCreateRequest = {
 export type QuestionsSetKeyResponse = {
     questionsSetKey: string
 }
+
+export type TeacherQuestionResponse = {
+    question: string,
+    correct: string,
+    answers: string[]
+}
+
+export type QuizFullResponse = {
```

Comment:
I would rename this `QuizFullResponse` type to `QuestionsSetResponse` or sth like that

Reply:
Done.